### PR TITLE
V20.07.x

### PR DESCRIPTION
**Pull request by gila**

### CODE REVIEW

File: mayastor/src/target/iscsi.rs [hunk 1]
```diff
@@ -165,51 +165,45 @@ fn destroy_iscsi_groups() {
...
+            &mut bdev.name().into_cstring().as_ptr(),
+            lun_ids,
+            1,
+            128,
+            true,
+            false,
+            false,
+            0,
+            false,
+            false,
```
Comment (jonathan-teh) [inline mayastor/src/target/iscsi.rs:202]: Could the comments be preserved? It's less readable now.

File: nix/pkgs/libspdk/default.nix [hunk 2]
```diff
@@ -55,8 +64,8 @@ stdenv.mkDerivation rec {
     "${enableFeature enableDebug "unit-tests"}"
     "--target-arch=nehalem"
     "--without-isal"
+    "--without-vhost"
```
Comment (jonathan-teh) [inline nix/pkgs/libspdk/default.nix:67]: Presumably we don't use/need vhost at the moment, is there another reason this is being disabled?

File: mayastor/tests/common/error_bdev.rs [hunk 3]
```diff
@@ -22,7 +18,9 @@ pub fn create_error_bdev(error_device: &str, backing_device: &str) {
     let err_bdev_name_str = std::ffi::CString::new(error_device.to_string())
         .expect("Failed to create name string");
     unsafe {
-        retval = spdk_vbdev_error_create(err_bdev_name_str.as_ptr()); // create the error bdev around it
+        retval = vbdev_error_create(err_bdev_name_str.as_ptr()); // create the
```
Comment (tiagolobocastro) [inline mayastor/tests/common/error_bdev.rs:21]: nit: I'd suggest adding the comment as one line above

File: mayastor/src/target/iscsi.rs [hunk 4]
```diff
@@ -165,51 +165,45 @@ fn destroy_iscsi_groups() {
...
-            &mut lun_id as *mut _, // lun id
-            1,                     // length of lun id list
-            128,                   // max queue depth
-            false,                 // disable chap
-            false,                 // require chap
-            false,                 // mutual chap
-            0,                     // chap group
-            false,                 // header digest
-            false,                 // data digest
+            -1,
```
Comment (tiagolobocastro) [inline mayastor/src/target/iscsi.rs:187]: it'd be nice to keep/update the comments

File: mayastor/tests/common/error_bdev.rs (hunk 3 above)
Comment (gila) [reply mayastor/tests/common/error_bdev.rs:21]: I had that but rust-fmt disagrees.

File: mayastor/src/target/iscsi.rs (hunk 4 above)
Comment (gila) [reply mayastor/src/target/iscsi.rs:187]: (1) the comments are not checked and or updated (biggest issue)
(2) the function arguments are pretty self-explanatory 
(3) we don't do that anywhere else

File: mayastor/src/target/iscsi.rs (hunk 1 above)
Comment (gila) [reply mayastor/src/target/iscsi.rs:202]: We don't comment function arguments anywhere else and they have to be updated/maintained. IMHO it is better this way as it forces you to check/validate the prototype of the function and not just assume the comments are (still) correct.

**Pull request merged**